astropy/units/format/utils.py

```diff
@@ -7,10 +7,9 @@
 
 import warnings
 
+from astropy.units.utils import maybe_simple_fraction
```

Comment:
Not a big deal, but the order here seems not ideal -- this is still an import within the submodule, so more "local" than `astropy.utils`.